appcelerator/amp-docker-kafka
# longer timeout in healthcheck

- fix appcelerator/amp#322

how to test:
- remove all containers related to kafka
- start amp
- wait ~ 1mn
- check that only one kafka container has been launched: docker service ps kafka
